Source/Scene/Scene.js

```diff
@@ -2857,25 +2899,29 @@ define([
         }
 
         var useOIT = environmentState.useOIT;
-        var useFXAA = environmentState.useFXAA;
+        var usePostProcess = environmentState.usePostProcess;
+
+        var defaultFramebuffer = environmentState.originalFramebuffer;
+        var globeFramebuffer = useGlobeDepthFramebuffer ? scene._globeDepth.framebuffer : undefined;
+        var sceneFramebuffer = scene._sceneFramebuffer.getFramebuffer();
 
         if (useOIT) {
-            passState.framebuffer = useFXAA ? scene._fxaa.getColorFramebuffer() : undefined;
+            passState.framebuffer = usePostProcess ? sceneFramebuffer : defaultFramebuffer;
             scene._oit.execute(context, passState);
         }
 
-        if (useFXAA) {
-            if (!useOIT && useGlobeDepthFramebuffer) {
-                passState.framebuffer = scene._fxaa.getColorFramebuffer();
-                scene._globeDepth.executeCopyColor(context, passState);
-            }
+        if (usePostProcess && (useOIT || defined(globeFramebuffer))) {
+            var inputFramebuffer = useOIT ? sceneFramebuffer : globeFramebuffer;
 
-            passState.framebuffer = environmentState.originalFramebuffer;
-            scene._fxaa.execute(context, passState);
+            var postProcess = scene.postProcessStages;
+            var colorTexture = inputFramebuffer.getColorTexture(0);
+            var depthTexture = defaultValue(globeFramebuffer, sceneFramebuffer).depthStencilTexture;
+            postProcess.execute(context, colorTexture, depthTexture);
+            postProcess.copy(context, defaultFramebuffer);
```

Comment:
Should post process be able to accept an output framebuffer to avoid the copy step?